data/campaigns/Under_the_Burning_Suns/scenarios/02_Across_the_Harsh_Sands.cfg

```diff
@@ -1430,17 +1432,55 @@
             y=13-19
         [/filter]
 
+#ifdef EASY
+        [scroll_to]
+            x,y=10,14
+        [/scroll_to]
+#else
         [terrain]
             terrain=Dd
             x=10,10,11,11,11,12,12
             y=14,15,15,16,17,15,16
         [/terrain]
-
+#endif
         {CHECK_EXPLORER}
+#ifdef EASY
+        [message]
+            speaker=$explorer.id
+            message= _ "Phew, thank goodness that this oasis is here! After hearing about the bandits in this area, I was beginning to worry that they had created a mirage to trick thirsty travelers such as ourselves!"
+        [/message]
+        [message]
+            speaker=Nym
+            message= _ "I know these bandits are evil, but do you really think they'd stoop so low as to get their victims' hopes up before crushing them?"
```

Comment:
Typographic apostrophes needed.

Reply:
...right, fixed in [1a39c06](https://github.com/wesnoth/wesnoth/pull/8421/commits/1a39c060ce6fb1d7ec7d61d79e63d7c106a45a60).